### PR TITLE
API-12093-insbuffer-create

### PR DESCRIPTION
**Pull request by jhulbert-va**
# [API-12093](https://vajira.max.gov/browse/API-12093)

### CODE REVIEW

File: vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/coverage/R4SiteInsuranceBufferCoverageController.java [hunk 1]
```diff
@@ -22,10 +33,55 @@
 public class R4SiteInsuranceBufferCoverageController {
   private final R4BundlerFactory bundlerFactory;
 
+  private final CharonClient charon;
+
+  private final WitnessProtection witnessProtection;
+
+  private static String beneficiaryOrDie(Coverage body) {
```
Comment (ilaflamme-va) [inline vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/coverage/R4SiteInsuranceBufferCoverageController.java:40]: This feels wrong to be in the controller like this.

Comment (jhulbert-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/coverage/R4SiteInsuranceBufferCoverageController.java:40]: Definitely weird, but we need it to provide the patient icn to the context.

Comment (ilaflamme-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/coverage/R4SiteInsuranceBufferCoverageController.java:40]: alright, I'm fine with it

Comment (ilaflamme-va) [reply vista-fhir-query/src/main/java/gov/va/api/health/vistafhirquery/service/controller/coverage/R4SiteInsuranceBufferCoverageController.java:40]: :shipit:

**Pull request merged**